src/components/TodoMain.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoMain: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => (
+        <div className={(classNames('todo', {
```

Comment:
Don't forget about the `key` attribute.

Reply:
agree with [comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/682/files#r1276832686)